cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
```

Comment:
I am ok with a little complexity to print the message only when needed, this function is not very reusable without documenting the "protocol" to use it (i.e. need to listen on the returned channel, etc). Perhaps it can just be placed in the file where it is used, alternatively, for better reusability, I suggest:

1. making the returned channel - a "cancellation" channel - i.e. if we write to the channel the printing is cancelled (if it has not already happened) [rather than the current pattern of reading from it]
2. buffer the returned channel to size 1 - this makes sure that there is no unnecessary blocking when either sender/receiver is not listening on the returned channel - it makes sure that cancelling the print is an optional behaviour.